app/controllers/v0/sessions_controller.rb

```diff
@@ -125,12 +129,39 @@ def user_logout(saml_response)
       end
     end
 
-    def stats(status, saml_response = nil, failure_tag = nil)
+    # rubocop:disable Metrics/MethodLength
+    def login_stats(status, saml_response, user_session_form = nil)
       case status
       when :success
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY) if request_type == 'signup'
+        # track users who have a shared sso cookie
+        if cookies.key?(Settings.sso.cookie_name)
+          StatsD.increment(STATSD_LOGIN_SHARED_COOKIE,
+                           tags: ["loa:#{@current_user.loa[:current]}",
+                                  "idp:#{@current_user.identity.sign_in[:service_name]}"])
+        end
+        StatsD.increment(STATSD_LOGIN_STATUS,
+                         tags: ['status:success',
+                                "idp:#{@current_user.identity.sign_in[:service_name]}",
+                                "context:#{saml_response.authn_context}"])
```

Comment:
do we gain any additional information from having both `idp` and `context`? 

id.me context can be rather long (`http://idmanagement.gov/ns/assurance/loa/1/vets`) and has a colon in it, which I think may cause problems for statsd.

in current dashboards, context has values like  dslogon, idme, mfa, mhv, signup, slo, verify. not `authn_context`

Reply:
the goal was to know the IDP and the level the user is authenticating at (eg level 1 vs level 3).  If we only use the path and drop the `http:` prefix, would that work better with statsd?

Reply:
I was mistaken, they're fine as is 
like this graph...
http://grafana.vfs.va.gov/d/000000033/site-authentication?orgId=1&fullscreen&panelId=16